sandbox-web/src/main/java/com/exadel/sandbox/team5/DiscountRestController.java

```diff
@@ -30,6 +31,7 @@ public DiscountDto getDiscount(@PathVariable Long id) {
         return service.getById(id);
     }
 
+    //fixme delete this method?
```

Comment:
If we already don't use it on UI we can remove this one. We can return it at any time from Git history if it will be required.